frameworks/hdfs/src/main/dist/core-site.xml

```diff
@@ -652,19 +547,109 @@
     </property>
     <property>
         <name>hadoop.http.authentication.cookie.domain</name>
-        <value>{{HADOOP_HTTP_AUTHENTICATION_COOKIE_DOMAIN}}</value>
+        <value>{{FRAMEWORK_HOST}}</value>
     </property>
     <property>
         <name>hadoop.http.authentication.simple.anonymous.allowed</name>
-        <value>{{HADOOP_HTTP_AUTHENTICATION_SIMPLE_ANONYMOUS_ALLOWED}}</value>
+        <value>false</value>
     </property>
     <property>
         <name>hadoop.http.authentication.kerberos.principal</name>
-        <value>{{HADOOP_HTTP_AUTHENTICATION_KERBEROS_PRINCIPAL}}</value>
+        <value>{{KERBEROS_PRIMARY_HTTP}}/{{FRAMEWORK_NAME}}.marathon.autoip.dcos.thisdcos.directory@{{KERBEROS_REALM}}</value>
```

Comment:
Should this refer to the `marathon` subdomain here?

Reply:
No other dns endpoint resolves to a service-specific address, as everything else is task-specific.